contracts/TestContracts.sol

```diff
@@ -158,3 +159,15 @@ contract XC20Test is ERC20 {
         _burn(from, amount);
     }
 }
+
+contract ERC20PresetMinterPauser_Decimals is ERC20PresetMinterPauser {
```

Comment:
Minor nitpick but we don't have _ anywhere in names so probably best to keep it that way.